data/abilities.js

```diff
@@ -3529,7 +3529,7 @@ let BattleAbilities = {
 			if (effect && effect.id === 'toxicspikes') return;
 			if (status.id === 'slp' || status.id === 'frz') return;
 			this.add('-activate', target, 'ability: Synchronize');
-			source.trySetStatus(status, target, Object.assign({status: status.id}, this.effect));
+			source.trySetStatus(status, target);
```

Comment:
Synchronize is a primary effect so it needs to pass an object with the appropriate status to `trySetStatus` otherwise it's processed as if it's a secondary effect.

Reply:
@urkerab Can you explain the difference between "primary effect" and "secondary effect"? That's not a difference I remember implementing when I made the battle engine.